topi/python/topi/nn/elemwise.py

```diff
@@ -42,3 +43,48 @@ def _compute(*indices):
         calpha = tvm.const(alpha, value.dtype)
         return tvm.select(value > 0, value, value * calpha)
     return tvm.compute(x.shape, _compute)
+
+@tvm.tag_scope(tag=tag.ELEMWISE)
+def prelu(x, slope, layout='NCHW'):
```

Comment:
change to axis implementation to be consistent with c++

Reply:
The tag should be tag.BROADCAST, because it is not ELEMWISE with respect to slope